### PR TITLE
fix: cleaning up sso self service wording and behaviors

### PR DESCRIPTION
**Pull request by alex-sheehan-edx**
# For all changes

- [ ] Ensure adequate tests are in place (or reviewed existing tests cover changes)

# Only if submitting a visual change

- [ ] Ensure to attach screenshots
- [ ] Ensure to have UX team confirm screenshots

### CODE REVIEW

File: src/components/settings/SettingsSSOTab/tests/ExistingSSOConfigs.test.jsx [hunk 1]
```diff
@@ -104,12 +109,15 @@ describe('<ExistingSSOConfigs />', () => {
     );
     expect(screen.getByText('nacho cheese')).toBeInTheDocument();
     expect(screen.getByText('Inactive')).toBeInTheDocument();
-
-    userEvent.click(screen.getByTestId(`existing-sso-config-card-dropdown-${inactiveConfig[0].id}`));
+    act(() => {
+      userEvent.click(screen.getByTestId(`existing-sso-config-card-dropdown-${inactiveConfig[0].id}`));
+    });
```
Comment (marlonkeating) [inline src/components/settings/SettingsSSOTab/tests/ExistingSSOConfigs.test.jsx:114]: Why is act() needed here?  It was my understanding that @testing-library/react `userEvent` operations are already wrapped in act().

File: src/components/settings/SettingsSSOTab/tests/NewSSOConfigForm.test.jsx [hunk 2]
```diff
@@ -103,7 +111,11 @@ describe('SAML Config Tab', () => {
     render(
       <Provider store={store}>
         <SSOConfigContext.Provider value={contextValue}>
-          <Provider store={store}><NewSSOConfigForm enterpriseId={enterpriseId} /></Provider>
+          <NewSSOConfigForm
+            enterpriseId={enterpriseId}
+            setIsStepperOpen={jest.fn()}
```
Comment (marlonkeating) [inline src/components/settings/SettingsSSOTab/tests/NewSSOConfigForm.test.jsx:116]: nit: why aren't we using `setIsStepperOpen` as defined above?

**Review by marlonkeating — APPROVED** [review]

File: src/components/settings/SettingsSSOTab/tests/ExistingSSOConfigs.test.jsx (hunk 1 above)
Comment (alex-sheehan-edx) [reply src/components/settings/SettingsSSOTab/tests/ExistingSSOConfigs.test.jsx:114]: I was just following and fixing the warnings printed out by the tests, I believe it has to do with updates triggered by the click and not just the click itself? However I'm not entirely sure

File: src/components/settings/SettingsSSOTab/tests/NewSSOConfigForm.test.jsx (hunk 2 above)
Comment (alex-sheehan-edx) [reply src/components/settings/SettingsSSOTab/tests/NewSSOConfigForm.test.jsx:116]: uhhhh I think the mocked func came later when I was cleaning stuff up. I can backfill it

**Pull request merged**